test/table.ts

```diff
@@ -110,14 +110,21 @@ describe('Bigtable/Table', () => {
   let table: any;
 
   before(() => {
-    Table = proxyquire('../src/table.js', {
+    const FakeTabularApiSurface = proxyquire('../src/tabular-api-surface.js', {
       '@google-cloud/promisify': fakePromisify,
       './family.js': {Family: FakeFamily},
       './mutation.js': {Mutation: FakeMutation},
       './filter.js': {Filter: FakeFilter},
       pumpify,
       './row.js': {Row: FakeRow},
       './chunktransformer.js': {ChunkTransformer: FakeChunkTransformer},
+    }).TabularApiSurface;
+    Table = proxyquire('../src/table.js', {
+      '@google-cloud/promisify': fakePromisify,
+      './family.js': {Family: FakeFamily},
+      './mutation.js': {Mutation: FakeMutation},
+      './row.js': {Row: FakeRow},
+      './tabular-api-surface': {TabularApiService: FakeTabularApiSurface},
```

Comment:
still a couple "services" here

Reply:
Yup. I noticed the tests failing and then adjusted these.

Reply:
I'll have to investigate the reason the most recent tests are failing.

Reply:
It looks like there's still at least one :) 

(TabularApiService)

Reply:
Like you pointed out... It looks like `TabularApiService` still needs to be renamed.